core/common/src/main/java/alluxio/metrics/sink/CsvSink.java

```diff
@@ -105,4 +113,19 @@ private TimeUnit getPollUnit() {
     }
     return TimeUnit.valueOf(unit.toUpperCase());
   }
+
+  /**
+   * Create the path for CSV sink if target path is not created.
+   *
+   * @param path CSV target path
+   */
+  private void createPollDir(File path) {
+    if (!path.exists()) {
+      try {
+        path.mkdirs();
+      } catch (SecurityException e) {
+        LOG.warn("Fail to create path {} for CSV sink, {}", path.toString(), e.getMessage());
```

Comment:
nit: `toString()` is implied here

```suggestion
        LOG.warn("Fail to create path {} for CSV sink, {}", path, e.getMessage());
```